### ISSUE TITLE
travis behaving weird on long standing PR

### ISSUE THREAD

**Issue by joernhees** [travis-ci/travis-ci#7237]
see https://github.com/RDFLib/rdflib/pull/514

the confusing thing is that the build results differ:
- there's a build running directly on the pre-release branch 5.0.0-dev
- another runs for the PR into master

expectation is that they don't diverge

**Comment by BanzaiMan** [comment]
Please try updating the topic branch with the head branch and see how the builds behave.

**Comment by joernhees** [comment]
TBH i don't understand what that means... :( are you referring to the github interface or shall i rebase the 5.0.0-dev branch on something (which would be very ugly as there are merges from master into it))

**Comment by joernhees** [comment]
Yes, i reported this, because something in there was flaky. As pointed out in https://github.com/RDFLib/rdflib/pull/514#issuecomment-275270862 the build ran twice, once directly per spec on the 5.0.0-dev branch (https://travis-ci.org/RDFLib/rdflib/builds/195384703) and once on the PR (https://travis-ci.org/RDFLib/rdflib/builds/195376794). The PR one reliably fails :-/

As we just merged 5.0.0-dev into master, the PR is now closed and we don't have a problem with it anymore. Thanks for looking into this, i'll close this for now.
